### ISSUE TITLE
[Suggestion]: Non Elemental Dragon, new dragon that can choose from all skills

### ISSUE THREAD

**Issue by InkDragon** [DragonSurvivalTeam/DragonSurvival#217]
### ■ Your Discord ID

Ink Dragon#0303

### ■ Dragon Survival version

DragonSurvival_DragonEditor-1.18.2-1.4.22

### ■ Minecraft version

1.18

### ■ Single game or server?

Single (Default)

### ■ Do you delete old configs before writing a bug report?

Nope (Default)

### ■ Description of incompatibility

I'd like to suggest the addition of a new dragon. This dragon would allow the player to choose from all other dragons for both negative and positive abilities as well as from all visual appearances. 

basically just an option to play as a dragon without any restictions

### ■ To Reproduce

this is just a suggestion, there was no option for suggestions.

**Comment by Horeak** [comment]
It is also a very impractical suggestion when considering the technical implementation.

You can already achieve a somewhat similar result by turning off the penalties in the server config
